libraries/standard/mqtt/cbmc/proofs/MQTT_DeserializeAck/Makefile

```diff
@@ -0,0 +1,17 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=MQTT_DeserializeAck_harness
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROOF_SOURCES += $(PROOFDIR)/../../sources/mqtt_cbmc_state.c
```

Comment:
sources, not src?

Reply:
That is the layout from the CBMC templates. Changing it here would warrant a change in the templates.